islandora_solr.install

```diff
@@ -53,6 +53,8 @@ function islandora_solr_uninstall() {
   // Removing variables.
   $variables = array_merge(islandora_solr_search_settings_variables(), array(
     'islandora_solr_url',
+    'islandora_solr_advanced_search_block_lucene_syntax_escape',
+    'islandora_solr_advanced_search_block_lucene_charset_default,',
```

Comment:
Looks like there's a comma as the last character of the variable name, here... It was probably supposed to be outside the single-quotes originally,